docs/Installation/Build-From-Source.md

```diff
@@ -0,0 +1,99 @@
+description: Building Orion from source code
+<!--- END of page meta data -->
+
+# Build from Source
+
+## Prerequisites
+
+* [Java JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+
+!!!important
+    Orion requires Java 8+ to compile; earlier versions are not supported.
+
+* [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/)
+
+* [libsodium](Dependencies.md)
+
+## Installation on Linux / Unix / Mac OS X
+
+###Clone the Orion Repository
+
+Clone the **PegaSysEng/orion** repository:
+
+```bash
+$ git clone --recursive https://github.com/PegaSysEng/orion.git
+```
+
+### Build Orion
+
+After cloning, go to the `orion` directory.
+
+Build Orion with the Gradle wrapper `gradlew`, omitting tests as follows:
+
+```bash
+$ ./gradlew build -x test
```

Comment:
We have a installDist gradle task that we could easily use instead of having to unzip the artefact as we already advertise it for Pantheon doc https://docs.pantheon.pegasys.tech/en/latest/development/running-developer-builds/
Also we could do the same for Pantheon in the install from source page too.

Reply:
I'm going to raise a JI so we can update this for Pantheon and Orion at the same time.